workers/blobuploadcleanupworker/blobuploadcleanupworker.py

```diff
@@ -15,7 +15,7 @@
 
 logger = logging.getLogger(__name__)
 
-DELETION_DATE_THRESHOLD = timedelta(days=2)
+DELETION_DATE_THRESHOLD = timedelta(hours=12)
```

Comment:
Can we make this configurable instead. Something like `BLOB_UPLOAD_DELETION_DATE_THRESHOLD`.
This way we can keep the current value as default, but allow it to be shorter or longer for others.

Reply:
Added